src/bots/depositor_bot.py

```diff
@@ -269,36 +276,47 @@ def _high_gas_fee_issue(self) -> bool:
             })
             return True
 
-    def _prohibit_to_deposit_issue(self) -> bool:
-        try:
-            can_deposit = contracts.deposit_security_module.functions.canDeposit(MODULE_ID).call(block_identifier=self.current_block.hash.hex())
-            logger.info({'msg': 'Call canDeposit().', 'value': can_deposit})
-
-            if not can_deposit:
-                logger.warning({'msg': self.DEPOSIT_SECURITY_ISSUE, 'value': can_deposit})
-                return True
+    def _prohibit_to_deposit_issue(self, stacking_module_id: int) -> bool:
+        can_deposit = contracts.deposit_security_module.functions.canDeposit(stacking_module_id).call(
+            block_identifier=self.current_block.hash.hex(),
+        )
+        logger.info({'msg': 'Call canDeposit().', 'value': can_deposit})
 
-        # TODO: get rid of try catch after the V2 upgrade
-        except Exception as error:
-            logger.info({'msg': 'canDeposit call exception.', 'error': str(error)})
-            return False
```

Comment:
Is it work ok without this on mainnet? We added this check as in the old DSM contract the method signature does not contain a parameter `canDeposit()`

Reply:
As I see, we do not update old container and it works in parallel with new releases. So this update won't reach old version

Reply:
How will the error be handled? Will the bot constantly restart? We have tests for restarts in our infra and we can break something with this behavior